skills/csharp/emailskill/wwwroot/manifest/manifest-1.0.json

```diff
@@ -141,7 +141,10 @@
     "summaryResult": {
       "type": "object",
       "properties": {
-        "$ref": "#/definitions/actionResult",
+        "actionSuccess": {
+          "type": "bool",
```

Comment:
Use boolean.. I will create a pr to fix all of these

Reply:
fixed